README.md

```diff
@@ -2,6 +2,8 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses/branch/main/graph/badge.svg?token=LvEIQ6tYti)](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses)
 
+Heroku QA App Link: https://s22-4pm-2-courses-qa.herokuapp.com/
```

Comment:
Hi! In addition to adding the link to the deployed app on Heroku, can you also add a link to the Heroku dashboard?

[See relevant section of proj01 instructions](https://ucsb-cs156.github.io/s22/lab/proj01/#:~:text=Make%20an%20issue%2C%20branch%2C%20and%20PR%20to%20a%20link%20to%20the%20deployed%20app%20and%20a%20link%20to%20the%20Heroku%20dashboard%20into%20the%20README.md%20for%20the%20repo.)